### PR TITLE
doc: release-notes: Add changelog for ARM64, cache, IPC

### PR DESCRIPTION
**Pull request by carlocaione**
Add the changelog entries for ARM64, Cache API and IPC.

Signed-off-by: Carlo Caione <ccaione@baylibre.com>

### CODE REVIEW

File: doc/releases/release-notes-3.3.rst [hunk 1]
```diff
@@ -517,6 +523,14 @@ Libraries / Subsystems
     :kconfig:option:`CONFIG_REQUIRES_FULL_LIBC`, which automatically selects
     a compatible C++ standard library.
 
+* Cache
+
+  * Introduced new Cache API
+  * :kconfig:option:`CONFIG_HAS_ARCH_CACHE` has been renamed to
```
Comment (gmarull) [inline doc/releases/release-notes-3.3.rst:529]:
```suggestion
  * ``CONFIG_HAS_ARCH_CACHE`` has been renamed to
```

File: doc/releases/release-notes-3.3.rst [hunk 2]
```diff
@@ -517,6 +523,14 @@ Libraries / Subsystems
...
+    :kconfig:option:`CONFIG_ARCH_CACHE`
+  * :kconfig:option:`CONFIG_HAS_EXTERNAL_CACHE` has been renamed to
```
Comment (gmarull) [inline doc/releases/release-notes-3.3.rst:531]:
```suggestion
  * ``CONFIG_HAS_EXTERNAL_CACHE`` has been renamed to
```

File: doc/releases/release-notes-3.3.rst [hunk 3]
```diff
@@ -526,6 +540,14 @@ Libraries / Subsystems
     Kconfig option :kconfig:option:`CONFIG_FCB_ALLOW_FIXED_ENDMARKER`
     and setting the `FCB_FLAGS_CRC_DISABLED` flag in the :c:struct:`fcb` struct.
 
+* IPC
+
+  * Added ``ipc_rpmsg_deinit``, ``ipc_service_close_instance`` and
```
Comment (gmarull) [inline doc/releases/release-notes-3.3.rst:545]:
```suggestion
  * Added :c:func:`ipc_rpmsg_deinit`, :c:func:`ipc_service_close_instance` and
```

File: doc/releases/release-notes-3.3.rst [hunk 4]
```diff
@@ -526,6 +540,14 @@ Libraries / Subsystems
...
+    ``ipc_static_vrings_deinit``  functions
```
Comment (gmarull) [inline doc/releases/release-notes-3.3.rst:546]:
```suggestion
    :c:func:`ipc_static_vrings_deinit`  functions
```

**Review by gmarull — DISMISSED** [review]

**Review by stephanosio — DISMISSED** [review]

**Review by laurenmurphyx64 — DISMISSED** [review]

**Review by stephanosio — DISMISSED** [review]

**Comment by stephanosio** [comment]
@carlocaione This PR now has conflicts.

**Review by stephanosio — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

**Pull request merged**